config/versions.list

```diff
@@ -0,0 +1,5 @@
+created_at: 2015-08-23T17:22:53-07:00
```

Comment:
No, we're not including the real file in this repository. The file here should work for test/dev but the real file for staging/production will be placed on the servers via chef.